po/pt_BR.po

```diff
@@ -192,7 +192,7 @@ msgstr "pacotes ausentes no AUR: "
 
 #: src/download.rs:94
 msgid "marked out of date: "
-msgstr "desatualizados: "
+msgstr "marcado como desatualizado: "
 
 #: src/download.rs:105
 msgid "orphans: "
```

Comment:
Just a tiny suggestion, orphans as _órfãos_ in pt-br. Not sure if worth open a review just for this...